docs/reference-guides/providers/howto.md

```diff
@@ -33,6 +40,44 @@ Remember that most Cluster API Providers are upstream projects maintained by the
 ## Create Your Cluster Definition
 
 <Tabs>
+  <TabItem value="aws-rke2" label="AWS RKE2" default>
+
+
+Before creating an AWS+RKE2 workload cluster, it is required to build an AMI for the RKE2 version that is going to be installed on the cluster. You can follow the steps in the [RKE2 image-builder README](https://github.com/rancher/cluster-api-provider-rke2/tree/main/image-builder#aws) to build the AMI.
+
+We recommend you refer to the CAPRKE2 repository where you can find a [samples folder](https://github.com/rancher/cluster-api-provider-rke2/tree/main/samples/aws) with different CAPA+CAPRKE2 cluster configurations that can be used to provision downstream clusters. The [internal folder](https://github.com/rancher/cluster-api-provider-rke2/tree/main/samples/aws/internal) contains cluster templates to deploy an RKE2 cluster on AWS using the internal cloud provider, and the [external folder](https://github.com/rancher/cluster-api-provider-rke2/tree/main/samples/aws/external) contains the cluster templates to deploy a cluster with the external cloud provider.
+
+We will use the `internal` one for this guide, however the same steps apply for `external`.
+
+To generate the YAML for the cluster, do the following:
+
+1. Open a terminal and run the following:
+
+```bash
+export CLUSTER_NAMESPACE="cluster-namespace"
+export CLUSTER_NAME="cluster-name"
+export CABPR_CP_REPLICAS=3
+export CABPR_WK_REPLICAS=1
+export RKE2_VERSION=v1.26.0+rke2r1
+export AWS_NODE_MACHINE_TYPE=t3a.large
+export AWS_CONTROL_PLANE_MACHINE_TYPE=t3a.large 
+export AWS_SSH_KEY_NAME="aws-ssh-key"
+export AWS_REGION="aws-region"
+export AWS_AMI_ID="ami-id"
+
+clusterctl generate cluster \
+--from https://github.com/rancher/cluster-api-provider-rke2/blob/main/samples/aws/internal/cluster-template.yaml -n example-aws rke2-aws > aws-rke2-clusterctl.yaml
```

Comment:
Probably we need to decide on naming of the cluster to be used in examples of provider combinations or change auto import notes to make it clear on cluster name? That is because I noticed the auto import notes at the end of docs always expects the cluster name to be `cluster1`

Reply:
You are right @furkatgofurov7. This is a mistake coming from a different documentation source, I'll be fixing this.